src/mkdocs_autorefs/plugin.py

```diff
@@ -69,13 +123,23 @@ def get_item_url(self, anchor: str) -> str:
             KeyError: If there isn't an item by this identifier anywhere on the site.
         """
         try:
-            return self._url_map[anchor]
+            url = self._url_map[anchor]
         except KeyError:
+            try:
+                return self._get_inventory_item_url(anchor)  # Without applying relative_url
+            except KeyError:
+                pass
+
             new_anchor = self.get_fallback_anchor(anchor)
             if new_anchor and new_anchor in self._url_map:
-                return self._url_map[new_anchor]
+                url = self._url_map[new_anchor]
+
             raise
```

Comment:
Isn't this code always reaching the raise instruction? Making it useless?

Reply:
Oh yeah, it doesn't make sense with the last-minute changes I made